packages/googlechrome.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,55 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\sfta.ps1
+. $toolsPath\helpers.ps1
+
+$downloadDir = Get-PackageCacheLocation
+$installer          = 'googlechromeinstaller.msi'
+$packageArgs        = @{
+    packageName     = $env:ChocolateyPackageName
+    file            = Join-Path $downloadDir $installer
+    url             = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise.msi'
+    url64bit        = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise64.msi'
+    fileType        = 'MSI'
+    silentArgs      = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+    validExitCodes  = @(0)
+}
+
+# Download the installer
+$packageArgs['file'] = Get-ChocolateyWebFile @packageArgs
+
+Start-Sleep 2
+
+$sigValid = (Get-AuthenticodeSignature -FilePath $packageArgs['file']).Status -eq 'Valid'
+
+try {
+    if ($sigValid) {
+        $toolName = 'Google Chrome'
+        $category = 'Utilities'
+
+        VM-Write-Log "INFO" "`tSignature valid ... installing $category"
```

Comment:
```suggestion
        VM-Write-Log "INFO" "Signature valid ... installing $category"
```

I think we need to remove the tab in all messages. We use them to display a sub message of a related message displayed before.